doc/api/crypto.md

```diff
@@ -2800,6 +2800,47 @@ threadpool request. To minimize threadpool task length variation, partition
 large `randomFill` requests when doing so as part of fulfilling a client
 request.
 
+### `crypto.randomInt([min, ]max[, callback])`
+<!-- YAML
+added:
+  - REPLACEME
+-->
+
+* `min` {integer} Start of random range (inclusive). **Default**: `0`.
+* `max` {integer} End of random range (inclusive).
```

Comment:
No default?

Reply:
Yes, not sure what a sane default would be here. Better leave it to the user.

Reply:
Would the default not be the max safe integer or the max allowable range from min? Whichever is less?

Reply:
I don’t think using a default like that would be helpful in most situations

Reply:
Ok. Just being thorough. That's good enough for me :)